kubernetes/coredns.yaml.sed

```diff
@@ -15,6 +15,7 @@ rules:
   - ""
   resources:
   - endpoints
+  - endpointslices
```

Comment:
We might need to add `endpointslices` in a separate apigroup rule:
```
- apiGroups:
    - discovery.k8s.io
    resources:
    - endpointslices
    verbs:
    - list
    - watch
```

Reply:
yes thanks - i was just trying to find a reference for that.